lib/logitech_receiver/receiver.py

```diff
@@ -336,10 +336,14 @@ def __init__(self, handle, device_info):
 
 		# read the serial immediately, so we can find out max_devices
 		# this will tell us if it's a Unifying or Nano receiver
-		serial_reply = self.read_register(_R.receiver_info, 0x03)
-		assert serial_reply
-		self.serial = _strhex(serial_reply[1:5])
-		self.max_devices = ord(serial_reply[6:7])
+                if self.product_id != 'c534':
```

Comment:
This is an ugly workaround and there is problem with recognition of other receivers too. I am not comfortable with this workaround. Hence, I would like to see some comments on this regards. @Lekensteyn @pwr anyone else?